scripts/run_simulation.py

```diff
@@ -0,0 +1,121 @@
+import sys
+import os
+from io import BytesIO
+
+import yaml
+
+import numpy as np
+from mpi4py import MPI
+
+from rexfw.convenience import setup_default_re_master
+from rexfw.convenience import setup_default_replica
+from rexfw.slaves import Slave
+from rexfw.samplers.rwmc import RWMCSampler
+from rexfw.pdfs.normal import Normal
+from rexfw.communicators.mpi import MPICommunicator
+
+# gross hack to get around the issue of dependency on a module in a
+# subdirectory of a Git repositry
+# (https://github.com/python-poetry/poetry/pull/2242)
+sys.path.append(os.getenv('RESAAS_COMMON_PATH',
+                          '/home/simeon/projects/resaas/common/resaas/'))
+from common.util import storage_factory
+
+# communicators are classes which serve as an interface between, say, MPI
+# and the rexfw code  other communicators could use, e.g., the Python
+# multiprocessing module to communicate between the master and the replicas
+
+# TODO: getting the number of replicas (and that's all the MPI stuff here is
+# for) should rather be a task of the communicator.
```

Comment:
Since we're wrapping this in calls to runmpi anyways, I don't think this is too much of an issue for the time being. If we later update the communication logic, etc. then we can always update how the number of replicas are specified.

Reply:
I agree. I think this was just a note to myself that it would be nicer if that logic was in the communicator.